lib/Zend/Validate/Hostname.php

```diff
@@ -565,14 +902,20 @@ class Zend_Validate_Hostname extends Zend_Validate_Abstract
         'today',
         'tokyo',
         'tools',
+        'top',
+        'toray',
+        'toshiba',
+        'tours',
         'town',
         'toys',
-        'tp',
```

Comment:
for the case someone wants to know why it got removed
source: https://en.wikipedia.org/wiki/.tp